test/commands/auth/device/login.test.ts

```diff
@@ -88,6 +87,9 @@ describe('auth:device:login', async () => {
     }
 
     stubMethod($$.SANDBOX, AuthInfo, 'create').callsFake(async () => authInfoStub);
+    $$.SANDBOX.stub(AuthInfo, 'listAllAuthFiles').callsFake(async () => {
+      return [`${authFields.username}.json`];
+    });
```

Comment:
Don't have to change it, but in these cases I think using `resolves()` instead of `callsFake()` is preferable.

Reply:
@shetzel I tried resolves, but could not get it to visit the mock, so I switched to callsFake.